apps/deck-playground/layers/fourwings/FourwingsHeatmapTileLayer.ts

```diff
@@ -29,6 +29,7 @@ export type FourwingsHeatmapTileLayerProps = {
   resolution?: FourwingsLayerResolution
   minFrame: number
   maxFrame: number
+  zOrderIndex: number
```

Comment:
What about making this prop optional? so we can use by default the `GROUP_ORDER` defined in the const but allow to override if needed.